models/cam-fv/README.rst

```diff
@@ -1,2724 +1,898 @@
-when needed, is not being actively supported for these CESM versions.
-<LI>Use either the eulerian, finite-volume (FV), or spectral-element (SE) dynamical core.</LI>
-<LI>Use any resolution of CAM, including refined mesh grids in CAM-SE.
-As of April, 2015 this is limited by the ability of the memory of a node of your hardware to contain 
-the state vector of a single ensemble member.  Work is under way to relax this restriction.
-</LI>
-<LI>Assimilate a variety of observations; to date the observations
-successfully assimilated include the NCEP reanalysis BUFR obs (T,U,V,Q), 
-Global Positioning System radio occultation obs, and MOPITT carbon monoxide 
-(when a chemistry model is incorporated into CAM-FV). 
-Research has also explored assimilating surface observations, 
-cloud liquid water, and aerosols.
-SABER and AURA observations have been assimilated into WACCM.</LI>
-<LI>Specify, via namelist entries, the CAM (initial file) variables which will be 
-directly affected by the observations, that is, the state vector.
-This allows users to change the model state without recompiling
-(but other restrictions remain).</LI>
-<LI>Generate analyses on the CAM grid which have only CAM model error in them, 
-rather than another model's.</LI>
-<LI>Generate such analyses with as few as 20 ensemble members.</LI>
-</UL>
-
-<P>
+
+-  Assimilate within the CESM software framework by using the multi-instance
+   capability of CESM1.1.1 (and later). This enables assimilation of suitable
+   observations into multiple CESM components. The ability to assimilate in the
+   previous mode, where DART called 'stand-alone' CAMs when needed, is not being
+   actively supported for these CESM versions.
+-  Use either the eulerian, finite-volume (FV), or spectral-element (SE)
+   dynamical core.
+-  Use any resolution of CAM, including refined mesh grids in CAM-SE. As of
+   April, 2015 this is limited by the ability of the memory of a node of your
+   hardware to contain the state vector of a single ensemble member. Work is
+   under way to relax this restriction.
+-  Assimilate a variety of observations; to date the observations successfully
+   assimilated include the NCEP reanalysis BUFR obs (T,U,V,Q), Global
+   Positioning System radio occultation obs, and MOPITT carbon monoxide (when a
+   chemistry model is incorporated into CAM-FV). Research has also explored
+   assimilating surface observations, cloud liquid water, and aerosols. SABER
+   and AURA observations have been assimilated into WACCM.
+-  Specify, via namelist entries, the CAM (initial file) variables which will be
+   directly affected by the observations, that is, the state vector. This allows
+   users to change the model state without recompiling (but other restrictions
+   remain).
+-  Generate analyses on the CAM grid which have only CAM model error in them,
+   rather than another model's.
+-  Generate such analyses with as few as 20 ensemble members.
```

Comment:
We need to get Kevin involved here ... I believe the Manhattan-compliant model_mod.f90 only supports the FV core.